src/bundlers/node/lambdaHandlerGenerator.ts

```diff
@@ -238,6 +238,10 @@ if (!genezioClass) {
                 });
             });
 
+            if(body.params && body.params.length > 0 && body.params[0].isGnzContext === true ) {
```

Comment:
We have a handler for runtime also. Should we put this also there?

Reply:
im not sure how the event coming from runtime looks like and i don't know if it's gonna change in the future so i don't know if we should put it just yet